composer.json

```diff
@@ -36,6 +36,12 @@
     ]
   },
   "extra": {
-    "altis": {}
+    "altis": {
+      "install-overrides": [
+        "humanmade/post-cloner",
+        "roots/wp-password-bcrypt",
```

Comment:
This one isn't a WordPress package type so doesn't need special handling, still won't do any harm though.

Reply:
My bad; tried to cross-ref against altis/core, but accidentally picked up another one by the looks of it.